### PR TITLE
changed the intro to recommend installing from release tar.

### PR DESCRIPTION
**Pull request by ttkorhonen**
Tried to address issue #70 . Please review.

### CODE REVIEW

File: getting-started/installation-linux.rst
Comment (ralphlange) [inline getting-started/installation-linux.rst:25]: release packages

File: getting-started/installation-linux.rst
Comment (ralphlange) [inline getting-started/installation-linux.rst:35]: cd base-7.0.8.1

**Review by ralphlange — CHANGES_REQUESTED** [review]
Please fix the two small issues before merging.

File: getting-started/installation-linux.rst
Comment (anjohnson) [inline getting-started/installation-linux.rst:26]: I don't see the need to mention submodules here, maybe instead use

> The released versions of EPICS have been fully tested to work as documented.

**Review by anjohnson — COMMENTED** [review]
One minor tweak, otherwise I'm fine with this.

**Comment by ralphlange** [comment]
If someone complains about that, send them to me...

**Comment by ttkorhonen** [comment]
OK then! For once, documentation is ahead of implementation.

**Comment by anjohnson** [comment]
Oh yes, [that](https://github.com/epics-base/epics-base/pull/491) was another thing I was in the middle of doing wasn't it. Nobody has actually approved my PR yet though, and I believe this will be our first release where the tagged release commit won't be an ancestor of all the commits that were merged after I branched but before that gets merged. That will be an interesting merge...

**Comment by ralphlange** [comment]
> Oh yes, [that](https://github.com/epics-base/epics-base/pull/491) was another thing I was in the middle of doing wasn't it. Nobody has actually approved my PR yet though,

Didn't you mark that PR as "don't ever merge"? Why approve something when there's no intention to merge?
Sorry, but your procedure is as unusual as unclear.

> and I believe this will be our first release where the tagged release commit won't be an ancestor of all the commits that were merged after I branched but before that gets merged. That will be an interesting merge...

*Shrug.*
Your procedure, your rules.

**Comment by anjohnson** [comment]
@ralphlange I was hoping for some feedback about the proposal, I'd rather not assume that no response from you guys means approval. The warning at the top was to ensure nobody else would actually merge the PR as committed, which would have caused a mess.

**Pull request merged**
